server/routes/Categories.js

```diff
@@ -26,7 +26,7 @@ router.delete('/:categoryID', async (req, res) => {
       return res
         .status(409)
         .send(
-          'The category is still used by other TODO Items. So, the category cannot be deleted'
+          'This category is still in use and so cannot be deleted.'
```

Comment:
![image](https://github.com/awctw/study-dash/assets/56493743/9cd7ee19-23d4-4135-98b9-b193d2f766f1)

If you're using the syntax I previously suggested above there is a typo here in the code (missing "it").

Reply:
I think that the sentence I have now is grammatically correct. Can you point out why it is not grammatically correct?

Reply:
Looks like it depends on who you ask:
![image](https://github.com/awctw/study-dash/assets/56493743/46d4be8d-ec66-4332-aac1-c120e0d8cfbd)

![image](https://github.com/awctw/study-dash/assets/56493743/0081e2f0-72a0-4b54-ac06-cae76f3fbc69)

Up to you if you want to make this change.

Reply:
I guess we can keep it the way it is, since the current version is the shortest of them call while looking grammatically correct?